packages/block-editor/src/components/global-styles/get-block-css-selector.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import { get, isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { scopeSelector } from './utils';
+
+export function getBlockCSSSelector(
+	blockType,
+	target = 'root',
+	options = {}
+) {
```

Comment:
Is there a reason that `options` part of the function signature is different from the PHP one?

```php
/**
 * Determine the CSS selector for the block type and property provided,
 * returning it if available.
 *
 * @since WordPress 6.3.0
 *
 * @param WP_Block_Type $block_type The block's type.
 * @param string|array  $target     The desired selector's target, `root` or array path.
 * @param boolean       $fallback   Whether or not to fallback to broader selector.
 *
 * @return string|null CSS selector or `null` if no selector available.
 */
function wp_get_block_css_selector( $block_type, $target = 'root', $fallback = false )
```

Reply:
Also, it would be useful to add the docblock here too.

```suggestion
/**
 * Determine the CSS selector for the block type and property provided,
 * returning it if available.
 *
 * @param {import('@wordpress/blocks').Block} blockType The block's type.
 * @param {string|string[]} target The desired selector's tartet, `root` or array path.
 * @param {object} options Options object.
 * @param {boolean} options.fallback Whether or not to fallback to broader selector.
 *
 * @returns {?string} The CSS selector or `null` if no selector available.
 */
export function getBlockCSSSelector(
	blockType,
	target = 'root',
	options = {}
) {
```

Reply:
✅ Done via separate commit to fix linting issues beforehand https://github.com/WordPress/gutenberg/pull/49393/commits/1ff98a84b5a3bcf027a71f855bf9b0bd876715e9

Thanks.

> Is there a reason that options part of the function signature is different from the PHP one?

This was a hangover from when I'd started these fixes before removing the editor-only selectors, where options also contained a `context` property.

With the potential to expand the API and reintroduce editor-only selectors, should we leave this options object? If leaving it, should the global PHP function be updated to use a similar options object?

Reply:
Would bringing consistency between the JS and PHP functions be ok as a follow up after the 15.5 RC has been cut or do we need a decision today? Currently `getBlockCSSSelector` is only exported via the private Global Styles APIs.